File: Programming Basics with JS/T04 - Nested Conditional Statements/exercise/06. Operations Between Numbers.js
function operationBetweenNums(input) {
    let N1 = Number(input[0]);
    let N2 = Number(input[1]);
    let op = input[2];
    let result = 0;

    switch(op){
        case "+":
            result = N1 + N2
            if (result % 2 === 0) {
                console.log(`${N1} ${op} ${N2} = ${result} - even`);
            } else {
                console.log(`${N1} ${op} ${N2} = ${result} - odd`);
            }
            break;

        case "-":
            result = N1 - N2
            if (result % 2 === 0) {
                console.log(`${N1} ${op} ${N2} = ${result} - even`);
            } else {
                console.log(`${N1} ${op} ${N2} = ${result} - odd`);
            }
            break;

        case "*":
            result = N1 * N2
            if (result % 2 === 0) {
                console.log(`${N1} ${op} ${N2} = ${result} - even`);
            } else {
                console.log(`${N1} ${op} ${N2} = ${result} - odd`);
            }
            break;

        case "/":
            if (N2 === 0) {
                console.log(`Cannot divide ${N1} by zero`);
            } else {
                result = N1 / N2
                console.log(`${N1} ${op} ${N2} = ${result.toFixed(2)}`);
            }
            break;

        case "%":
            if (N2 === 0) {
                console.log(`Cannot divide ${N1} by zero`);
            } else {
                result = N1 % N2
                console.log(`${N1} ${op} ${N2} = ${result}`);
            }
            break;
    }
}

File: Programming Basics with Python/T04 - Conditional Statements (Advanced)/exercise/06. Operations Between Numbers.py
n1 = int(input())
n2 = int(input())
operator = input()
result = 0
even_odd = ""

if operator == "/" and n2 == 0:
    print(f"Cannot divide {n1} by zero")

elif operator == "%" and n2 == 0:
    print(f"Cannot divide {n1} by zero")

elif operator == "+":
    result = n1 + n2
    if result % 2 == 0:
        even_odd = "even"
    else:
        even_odd = "odd"
    print(f"{n1} {operator} {n2} = {result} - {even_odd}")

elif operator == "-":
    result = n1 - n2
    if result % 2 == 0:
        even_odd = "even"
    else:
        even_odd = "odd"
    print(f"{n1} {operator} {n2} = {result} - {even_odd}")

elif operator == "*":
    result = n1 * n2
    if result % 2 == 0:
        even_odd = "even"
    else:
        even_odd = "odd"
    print(f"{n1} {operator} {n2} = {result} - {even_odd}")

elif operator == "/":
    result = n1 / n2
    print(f"{n1} / {n2} = {result:.2f}")

elif operator == "%":
    result = n1 % n2
    print(f"{n1} % {n2} = {result}")